src/RpcClient/RpcClient.cs

```diff
@@ -459,6 +450,43 @@ public async Task<RpcUnclaimedGas> GetUnclaimedGasAsync(string address)
             return RpcUnclaimedGas.FromJson(result);
         }
 
+        /// <summary>
+        /// Returns part results from Iterator.
+        /// This RPC call does not affect the blockchain in any way.
+        /// </summary>
+        public async Task<JArray> TraverseIteratorAsync(string sessionId, string id, int count)
+        {
+            var result = await RpcSendAsync(GetRpcName(), sessionId, id, count).ConfigureAwait(false);
+            return result as JArray;
+        }
+
+        /// <summary>
+        /// Returns all results from Iterator.
+        /// This RPC call does not affect the blockchain in any way.
+        /// </summary>
+        public async IAsyncEnumerable<JObject> TraverseIteratorAllAsync(string sessionId, string id, int count)
```

Comment:
Merge it with TraverseIteratorAsync?

Reply:
Which result type should the merged method use, JArray or IAsyncEnumerable?

Reply:
`IAsyncEnumerable`